### PR TITLE
ValueChecks: new methods - allTrue and allFalse

### PR DESCRIPTION
**Pull request by adamvoncorswant**
To make some statements more readable, for example:
```java
boolean passesSanityChecks = true;
passesSanityChecks &= clearedAccount.getCurrencyId() == liquidAccount.getCurrencyId();
passesSanityChecks &= clearedAccount.getOrganizationId().equals(liquidAccount.getOrganizationId());
passesSanityChecks &= clearedAccount.getAccountType() == AccountType.CLIENT_CLEARED_ACCOUNT;
passesSanityChecks &= liquidAccount.getAccountType() == AccountType.CLIENT_LIQUID_ACCOUNT;
passesSanityChecks &= originatingTxId != null;
if (!passesSanityChecks) {
    ....
}

```
can now be written instead as:
```java
final boolean passesSanityChecks = ValueChecks.allTrue(
    clearedAccount.getCurrencyId() == liquidAccount.getCurrencyId(),
    clearedAccount.getOrganizationId().equals(liquidAccount.getOrganizationId()),
    clearedAccount.getAccountType() == AccountType.CLIENT_CLEARED_ACCOUNT,
    liquidAccount.getAccountType() == AccountType.CLIENT_LIQUID_ACCOUNT,
    originatingTxId != null
);
if (!passesSanityChecks) {
    ....
}
```

### CODE REVIEW

**Comment by adamvoncorswant** [comment]
ping @mikaellothman

File: izettle-java/src/main/java/com/izettle/java/ValueChecks.java [hunk 1]
```diff
@@ -43,6 +43,30 @@ public static boolean allNull(Object... objects) {
         return true;
     }
 
+    public static boolean allFalse(boolean... tests) {
```
Comment (mikaellothman) [inline izettle-java/src/main/java/com/izettle/java/ValueChecks.java]: You could forces "at least one argument" by setting the parameter list to `allFalse(boolean value, boolean... values)` if you want to...

Comment (adamvoncorswant) [inline izettle-java/src/main/java/com/izettle/java/ValueChecks.java]: Good idea! BRB

**Comment by adamvoncorswant** [comment]
Discussed IRL. Closing.

**Pull request closed without merging**